Source/ACE.Server/Managers/HouseManager.cs

```diff
@@ -174,6 +174,7 @@ public static async void HandleEviction(PlayerHouse playerHouse)
 
             house.HouseOwner = null;
             house.MonarchId = null;
+            house.HouseOwnerName = null;
```

Comment:
not a huge deal, but is the overhead of storing HouseOwnerName really worth it here? is there anything to be gained, aside from "that's how retail did it"?

the house owner name is already accessible through PlayerManager.FindByGuid(Hook.House.HouseOwner).Name, and storing things statically here just leaves more room for error and possible bugs. for example, what happens if the player changes their name on the server? all references to their name should be updated to reflect that

retail stored a ton of runtime info in properties for many reasons, a lot were related to the technical implementation of the original servers. anytime anything could potentially be cross-landblock or cross-server (which i would imagine would be most things, ie. the possibility of appraising a housing hook from across a landblock boundary) retail had to store a ton of serialized info in runtime properties for inter-landblock and inter-server communication. it made sense in that situation, but for the current architecture, that same design decision might not apply

Reply:
I figured the cost of storing one text field outweighed the cost needed to load an "OfflineCharacter" when all we needed was just to pull a name.

Reply:
technically they are both dictionary lookups (looking up a string property vs. looking up an offline character) so it should take the same amount of processing time

the server keeps a dictionary of all the offline characters loaded on the server. its just returning a reference that has been pre-loaded. keeping fast lookup tables of all the offline characters is important for things like building the allegiance chain structures, so this process should be very efficient

again not a huge issue, i'm fine with the pr either way